dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -1,28 +1,27 @@
 # For testing and analyzing code.
-mypy~=0.701.0
-pylint~=2.3.0
-pytest~=3.8.0
-pytest-asyncio~=0.10.0
-pytest-cov~=2.5.0
-pytest-benchmark~=3.2.0
-yapf~=0.27.0
+mypy~=0.750
+pylint~=2.4
+pytest~=5.3
+pytest-asyncio~=0.10
+pytest-cov~=2.8
+pytest-benchmark~=3.2
+yapf~=0.29
```

Comment:
What kinds of changes to formatting does this cause?

Reply:
Also, this change is wrong because it means that yapf making changes will break our build.